docs/topics/db/queries.txt

```diff
@@ -1966,6 +1966,41 @@ relationships accept primary key values. For example, if ``e1`` and ``e2`` are
     a.entry_set.set([e1, e2])
     a.entry_set.set([e1.pk, e2.pk])
 
+Filters on RelatedManager
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When calling ``filter()`` on a
```

Comment:
Curious, does `exclude()` behave the same? We have one weirdness doc'd here:

https://github.com/django/django/blob/a1ce852e52f45da7528446cef3a1f02416531771/docs/topics/db/queries.txt#L660-L664